Documentation/how-to-build-and-run-ilcompiler-in-vscode.md

```diff
@@ -0,0 +1,58 @@
+_Please ensure that [pre-requisites](prerequisites-for-building.md) are installed for a successful build of the repo._
+
+_Note_:
+
+* Instructions below assume ```~/corert``` is the repo root.
+
+# Setting up #
+
+Please make sure you have latest VS Code, C# extension, and .NET Core available. This guide is tested under C# 1.6.2 + VS Code 1.8.1 + CLI 1.0.0-preview4-004233.
+
+This guide assumes that your VS code workspace is set to the root of the repo. 
+
+# Running VS Code
+
+We've checked-in reasonable default ```launch.json``` and ```tasks.json``` under ```corert/.vscode``` directory. You only need to run vscode form corert root:
```

Comment:
Nit: I would just say: launch.json and tasks.json with reasonable defaults are in corert/.vscode directory.